### ISSUE TITLE
Wiki Page Unclear on Musl Repo Location

### ISSUE THREAD

**Issue by JeffCarpenter** [ssrg-vt/hermitux#8]
Hi all,

I was browsing the wiki when I found the [Fast system calls in dynamically linked programs](https://github.com/ssrg-vt/hermitux/wiki/Fast-system-calls-in-dynamically-linked-programs) page. It looks pretty straight-forward otherwise, but it's not clear where to find the musl repo (though I did track it down via Google eventually).

Very cool project btw -- definitely will be keeping an eye on this one!

**Comment by olivierpierre** [comment]
Hello! thank you very much for the feedback!

I added some precision on the wiki page -- the `musl` directory used to be downloaded automatically as part of a bootstrap process for the project launched thorough a script. I have now changed this to use (1) submodules and (2) a regular Makefile at the root of the project.

Long story short, if the `musl` directory is not there, just type `make` once in the root directory. I hope it makes things more clear now :)
